haydn.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Vinayak",
+			"oem": "Xiaomi",
+			"device": "Mi 11X Pro",
+			"filename": "crDroidAndroid-14.0-20231115-haydn-v10.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/haydn/10.x/crDroidAndroid-14.0-20231115-haydn-v10.0-BETA.zip/download",
+			"timestamp": 1700015378,
+			"md5": "3d6ab3a49d4622d2be54afacd700aab3",
+			"sha256": "78cf9abd18fed30a2f6cc42f04ea82139c1ec5f84c136eba4dec6bd26777919d",
+			"size": 1758312963,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-official-crdroid-for-haydn-haydnin.4606329/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-U/07-Nov-2023/NikGapps-core-arm64-14-20231107-signed.zip/download",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "",
+			"paypal": "",
+			"telegram": "https://t.me/pandemonium_haydn",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_haydn",
```

Comment:
no longer valid
https://github.com/crdroidandroid/android_device_xiaomi_haydn/blob/14.0/vendorsetup.sh

if you setup crdroid.dependencies, remove lineage variant (cleanup)

Reply:
I have remove the the following from tree :

• lineage.dependenvies 
• vendorsetup.sh